backend/Squirrel.Core/Squirrel.Core.BLL/Services/ProjectService.cs

```diff
@@ -106,17 +121,19 @@ public async Task<List<UserDto>> GetProjectUsersAsync(int projectId)
     public async Task<List<ProjectResponseDto>> GetAllUserProjectsAsync()
     {
         var currentUserId = _userIdGetter.GetCurrentUserId();
-        var userProjects = await _context.Projects
+        var userProjects = await _context.Users
+            .Where(user => user.Id == currentUserId)
+            .SelectMany(user => user.Projects)
```

Comment:
User entity has two properties: Projects (for projects, where he is a member) and OwnProjects (projects, where he is an author). So consider searching all user's projects this way:
```csvar currentUserId = _userIdGetter.GetCurrentUserId();
        var userProjects = await _context.Projects
                                         .Include(p => p.Tags)
                                         .Where(p => p.CreatedBy == currentUserId ||
                                                     p.Users.Any(u => u.Id == currentUserId))
                                         .ToListAsync();```